src/shared/ui/survey/ParagraphText.tsx

```diff
@@ -16,15 +22,41 @@ type Props = {
 
 const ParagraphText: FC<Props> = ({ value, onChange, config, ...props }) => {
   const [paragraphOnFocus, setParagraphOnFocus] = useState(false);
+  const [keyboardHeight, setKeyboardHeight] = useState(0);
   const { maxLength = 50 } = config;
   const { t } = useTranslation();
 
+  useEffect(() => {
+    if (process.env.NODE_ENV === 'test') {
+      return;
+    }
+    if (Platform.OS != 'ios') {
+      const keyboardDidShowListener = Keyboard.addListener(
+        'keyboardDidShow',
+        event => {
+          setKeyboardHeight(event.endCoordinates.height * 0.8);
```

Comment:
Where does `0.8` come from? Can we create a const to avoid or explain magic numbers

Reply:
0.8 is 80% of keyboard size,  its enought to make user able to scroll, but I cab remove, the both will work.
what do you think better ?